src/OpenLoco/src/Drawing/SoftwareDrawingContext.cpp

```diff
@@ -1509,16 +1510,129 @@ namespace OpenLoco::Drawing
         }
 
         // 0x004474BA
+        // 0x00494E33
```

Comment:
not sure where this has come from thats a string function. could you remove the extra offset.